wasm/src/main/scala/ir2wasm/WasmExpressionBuilder.scala

```diff
@@ -1908,7 +1908,26 @@ private class WasmExpressionBuilder private (
 
       // Create the array object
       instrs += STRUCT_NEW(TypeIdx(WasmStructTypeName(arrayTypeRef)))
-    } else ??? // TODO support multi dimensional arrays
+    } else {
+      /* There is no Scala source code that produces `NewArray` with more than
```

Comment:
[note]
`Array.ofDim` just creates nested Arrays with `new Array(new Array...)`, and it doesn't create an Array with multiple lengths in Scala.js IR